hpx/components/containers/partitioned_vector/partitioned_vector_segmented_iterator.hpp

```diff
@@ -521,6 +521,11 @@ namespace hpx
           : base_type(it), data_(data)
         {}
 
+        operator BaseIter const &() const
```

Comment:
The template parameter `BaseIter` of `segment_vector_iterator` seems to be private and `segmented_iterator_traits` seems to not contain any `typedef` corresponding to that type.